commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/sftp/test/SftpProviderStreamProxyModeTestCase.java

```diff
@@ -0,0 +1,88 @@
+package org.apache.commons.vfs2.provider.sftp.test;
+
```

Comment:
ASL header is missing causing the build to fail on Travis.

Reply:
added